samples/server.c

```diff
@@ -461,7 +462,11 @@ int main(int argc, char *argv[])
         err(EXIT_FAILURE, "failed to register signal handler");
     }
 
-    vfu_ctx = vfu_create_ctx(VFU_TRANS_SOCK, argv[optind], 0, &server_data,
+    if (strcmp(argv[optind], "pipe") == 0) {
```

Comment:
Why do we support the pipe transport on the server but not on the client?

Reply:
afl is the client